python/orca/tutorial/pytorch/FashionMNIST/orca-tutorial-k8s-client.yaml

```diff
@@ -10,66 +10,56 @@ spec:
-                --conf spark.pyspark.driver.python=/bigdl/nfsdata/orca_env/bin/python \
-                --conf spark.pyspark.python=orca_env/bin/python \
-                --conf spark.executorEnv.PYTHONHOME=orca_env \
-                --conf spark.kubernetes.file.upload.path=/bigdl/nfsdata/ \
+                --archives file:///bigdl/nfsdata/environment.tar.gz#environment \
+                --conf spark.pyspark.driver.python=/bigdl/nfsdata/spark_submit_environment/bin/python \
+                --conf spark.pyspark.python=environment/bin/python \
                 --num-executors 2 \
-                --executor-cores 16 \
-                --executor-memory 50g \
-                --total-executor-cores 32 \
-                --driver-cores 4 \
-                --driver-memory 50g \
+                --executor-cores 4 \
+                --executor-memory 2g \
+                --total-executor-cores 8 \
+                --driver-cores 2 \
+                --driver-memory 2g \
                 --properties-file ${BIGDL_HOME}/conf/spark-bigdl.conf \
-                --py-files local://${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,local:///bigdl/nfsdata/train.py,local:///bigdl/nfsdata/model.py \
-                --conf spark.driver.extraJavaOptions=-Dderby.stream.error.file=/tmp \
+                --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,/bigdl/nfsdata/kai/train.py,/bigdl/nfsdata/kai/model.py \
                 --conf spark.kubernetes.executor.deleteOnTermination=True \
-                --conf spark.sql.catalogImplementation='in-memory' \
                 --conf spark.driver.extraClassPath=local://${BIGDL_HOME}/jars/* \
                 --conf spark.executor.extraClassPath=local://${BIGDL_HOME}/jars/* \
                 local:///bigdl/nfsdata/train.py
                 --cluster_mode spark-submit
-                --remote_dir file:///bigdl/nfsdata/dataset
+                --data_dir /bigdl/nfsdata/dataset
                 "]
         securityContext:
           privileged: true
         env:
           - name: RUNTIME_K8S_SPARK_IMAGE
-            value: intelanalytics/bigdl-k8s:2.1.0
+            value: intelanalytics/bigdl-k8s:latest
           - name: RUNTIME_SPARK_MASTER
             value: k8s://https://<k8s-apiserver-host>:<k8s-apiserver-port>
-          - name: RUNTIME_DRIVER_PORT
-            value: !!str 54321
           - name: SPARK_MODE
             value: client
-          - name: RUNTIME_K8S_SERVICE_ACCOUNT
-            value: spark
-          - name: BIGDL_HOME
-            value: /opt/bigdl-2.1.0
+          - name: SPARK_VERSION
+            value: 3.1.3
           - name: SPARK_HOME
             value: /opt/spark
-          - name: SPARK_VERSION
-            value: 3.1.2
           - name: BIGDL_VERSION
-            value: 2.1.0
+            value: 2.2.0-SNAPSHOT
+          - name: BIGDL_HOME
+            value: /opt/bigdl-2.2.0-SNAPSHOT
```

Comment:
Any better way not to hard code bigdl version here?

Reply:
May not?
Added some information in the container here: https://github.com/intel-analytics/BigDL/pull/7625/files#diff-e22baa5809a8b270b7a3145973bbe8f6504fc48001dd0898d018c871b616a3efR144